decentraland/unity-renderer
# Feat: Map render

## What does this PR change?



This pull request adds the new map renderer and the infinite floor with the map projected onto it.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/Map-Render
2. Go near the infinite floor and check it's being rendered properly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
